ckanext/password_policy/views.py

```diff
@@ -301,6 +314,7 @@ def logout():
 
 custom_user.add_url_rule(u'/locked', view_func=locked_user, methods=("GET", "POST"))
 custom_user.add_url_rule(u'/_logout', view_func=logout)
+custom_user.add_url_rule(u'/reset_login/<username>', view_func=reset_login, methods=("POST",))
```

Comment:
Small question, don't we need to use here also GET method? since we also need to get info in user before login reset

Reply:
no, the only info we need to reset the login is the username, which we get from the url already

Reply:
ok, i see, thank you :)